examples/dash_display_client_examples/battery_daughter.py

```diff
@@ -22,25 +23,54 @@
 
 ### WiFi ###
 
-# Get wifi details and more from a secrets.py file
-try:
-    from secrets import secrets
-except ImportError:
-    print("WiFi secrets are kept in secrets.py, please add them there!")
-    raise
-
+# Get wifi details and more from a settings.toml file
+# tokens used by this Demo: CIRCUITPY_WIFI_SSID, CIRCUITPY_WIFI_PASSWORD
+#                           CIRCUITPY_AIO_USERNAME, CIRCUITPY_AIO_KEY
+secrets = {}
+for token in ["ssid", "password"]:
+    if getenv("CIRCUITPY_WIFI_" + token.upper()):
+        secrets[token] = getenv("CIRCUITPY_WIFI_" + token.upper())
+for token in ["aio_username", "aio_key"]:
+    if getenv("CIRCUITPY_" + token.upper()):
+        secrets[token] = getenv("CIRCUITPY_" + token.upper())
```

Comment:
```suggestion
```suggestion
for token in ["SSID", "PASSWORD"]:
    if getenv("CIRCUITPY_WIFI_" + token):
        secrets[token] = getenv("CIRCUITPY_WIFI_" + token)
for token in ["AIO_USERNAME", "AIO_KEY"]:
    if getenv("CIRCUITPY_" + token):
        secrets[token] = getenv("CIRCUITPY_" + token)
```

Reply:
That's fine, I believe the secrets index will then need the .lower() modifier.

Regarding the examples that don't use the dash_display library (the ones in the dash_display_client_examples folder), I'm not all that familiar with adafruit.io so I didn't know if the client_examples interacted with dash_display_advancedtest.py. Unless someone that knows how adafruit.io/MQTT works looks them over and says they aren't related, I'd say we should just leave them as they are.

Reply:
I missed that `secrets` was used further down.

Reply:
It did actually occur to me that using the lower rather than two uppers was slightly more efficient :grin: